app/controllers/stories_controller.rb

```diff
@@ -15,6 +15,7 @@ class StoriesController < ApplicationController
 
   before_action :authenticate_user!, except: %i[index search show]
   before_action :set_cache_control_headers, only: %i[index search show]
+  # before_action :set_json_ld, only: :show
```

Comment:
Something I forgot to mention while we were pairing on this, @juliannatetreault: you might not even need to do this in a before action. Could just set the instance variable directly in the show action:

```
def show
  @json_ld = build_json_ld
end

private
def build_json_ld
  # all your fancy code to build that big object and call `to_json` on it goes here!
end

Reply:
That's not a bad idea; thanks! 😄